core/readme.md

```diff
@@ -1,86 +1,186 @@
-* Clarity of intent - new operations and functionality should be opinionated, and straightforward to use.
-* User safety over additional functionality - minimize attack surface at the lowest levels.
-* The Stellar network should run at scale and at low cost to all users.
-    * In support of this, the Stellar network should support off-chain transactions, e.g. Starlight.
-* The Stellar network should facilitate simplicity and interoperability with other protocols and networks.
-    * In support of this, the Stellar network should facilitate side-chain transactions to enable sub-networks.
-* The Stellar network should support decentralization wherever possible, but not at the expense of the majority of its values.
-* It should be easy to develop projects using the Stellar Network
-* The Stellar network should make it easy for developers of Stellar projects to create highly usable products
-* The Stellar network should enable cross-border payments, i.e. payments via exchange of assets, throughout the globe, enabling users to make payments between assets in a manner that is fast, cheap, and highly usable.
-    * In support of this, the Stellar network should support an orderbook that values simplicity over functionality, and one that primarily serves to enable cross-border payments.
-    * In support of this, the Stellar network should facilitate liquidity as a means to enabling cross-border payments.
-    * In support of this, the Stellar network should enable asset issuance, but as a means of enabling cross-border payments.
-* The Stellar network should enable users to easily exchange their non-Stellar based assets to Stellar-based assets, and vice versa.
-
-
+# Core Advancement Proposals (CAPs)
+
+## CAP Status Terms
+* **Archived** — A CAP that did not head towards a disposition due to a lack of consensus _and_
+  support. Generally open to revival with additional edits.
+* **Draft** — A CAP that is currently open for consideration and actively being discussed.
+* **Pending** — A CAP that is ready for final deliberation by the CAP Core Team. After a maximum of
+  three meetings, a vote will take place that will set the CAP's intended FCP disposition
+  (**FCP: Acceptance/Rejection**) or go back into a **Draft** state.
+* **FCP: [Acceptance/Rejection]** — A CAP that has entered a Final Comment Period (FCP) with an
+  intended disposition. After one week has passed, during which any new concerns should be
+  addressed, the CAP will head towards its intended disposition [**Acceptance/Rejection**] or go
+  back into a Draft state.
+* **Accepted** — A CAP that has been formally accepted and is ready for implementation. It is
+  expected to be included in a future version of the protocol.
+* **Finalized** — A CAP that has been implemented in Stellar Core in the version specified.
```

Comment:
What happens to rejected CAPs? `Archived`?

Reply:
Yeah I actually just changed it to `Rejected` or Perma-`Draft`. Basically if something doesn't ever get a decision or loses momentum will just stay as a draft.